CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = landed_gentleman
+		
+		any_pop = { 
+			ideology = { factor = 0.09 value = conservative }  
+		}
+		any_pop = { 
+			limit = { is_primary_culture = yes OR = { strata = rich strata = middle } }
+			ideology = { factor = 0.02 value = nationalist } 
+		}
+		any_pop = { limit = { 
+		    OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_763 region = SAR_765 } 
+			}
+			ideology = { factor = 0.04 value = traditionalist } 
+		}
+		any_pop = {
+			scaled_consciousness = { factor = 4 ideology = communist }
+			scaled_consciousness = { factor = 4 ideology = communist_social }
+			scaled_consciousness = { factor = 4 ideology = socialist }
+		}
+		
+		add_country_modifier = { name = political_strife duration = 365 }
+		
+		set_country_flag = ITA_Tambroni
+		
+		ai_chance = { factor = 0.9 }
+	}
+	
+	option = {
+		name = "Moro (DC-PSDI-PRI PSI)"
+		
+		ruling_party_ideology = big_tent
+		
+		#Leadership
+		remove_country_modifier = expert_diplomat
+		remove_country_modifier = successful_industrialist
+		remove_country_modifier = ivory_tower_intellectual
+		remove_country_modifier = man_of_the_people
+		remove_country_modifier = landed_gentleman
+		remove_country_modifier = military_man
+		remove_country_modifier = lord_admiral
+		remove_country_modifier = raving_loon
+		remove_country_modifier = luddite
+		remove_country_modifier = great_statesman
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = man_of_the_people 
+		
+		any_pop = { 
+			ideology = { factor = -0.05 value = conservative } 
+			ideology = { factor = 0.07 value = big_tent } 
+			ideology = { factor = 0.01 value = liberal } 
+		}
+		any_pop = { 
+			limit = { strata = middle }
+			ideology = { factor = 0.03 value = socialist }
+		}
+		any_pop = {
+			scaled_consciousness = { factor = 2 ideology = communist }
+		}
+		
+		add_country_modifier = { name = political_strife duration = 365 }
+		
+		set_country_flag = ITA_Democratic_Renovation
+		
+		ai_chance = { factor = 0.1 }
+	}
+   
+}
+
+#The Genoa Facts
+country_event = {
+  	id = 8020319
+  	title = "The Genoa Facts"
+  	desc = EVT_8020319_DESC
+  	picture = "rom_ethnic_clashes"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1960  
+		NOT = { year = 1963 }
+		
+		government = democracy
+		
+		ruling_party_ideology = conservative
+		
```

Comment:
I suppose this event relates to Genoa so please insert a check for ITA owning Genoa.